CRM/Membershipextrasdata/Upgrader.php

```diff
@@ -279,7 +279,7 @@ private function createIndividualMembershipPriceSet() {
 
     $createdRecordResponse = civicrm_api3('PriceSet', 'create', [
       'title' => 'Individual Membership',
-      'name' => 'FIELD2',
+      'name' => 'Individual_Membership',
```

Comment:
Should the name field use lower case? `'individual_membership'`?

Reply:
You are right, but there is no need because I will delete this commit and the helper `CRM_Membershipextrasdata_Utils_String::slugify` will take care of that after refactoring.